src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -610,6 +611,14 @@ protected override void HandleInitializeRequestAsync(IRequestResponder<Initializ
                 });
             }
 
+            List<ColumnDescriptor> additionalModuleColumns = new List<ColumnDescriptor>();
```

Comment:
You will want to only do this if the client is VS.
You can check this by:
`Host.GetHostUIIdentifier() == HostUIIdentifier.VSIDE`

Reply:
Does specifying `AdditionalModuleColumns` as blank do something unintended?

Reply:
Blank is fine.

Reply:
I lied earlier, I have updated the comment to do the correct check.